Utilities/Stop-Processes/Stop-Processes.psm1

```diff
@@ -1,102 +0,0 @@
-<#
```

Comment:
Why was this deleted?

Reply:
> `Stop-Process` is a cmdlet that is included with PowerShell (version core-6.1.0-windows) whose definition should not be overridden
> PSScriptAnalyzer([PSAvoidOverwritingBuiltInCmdlets](https://docs.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/rules/avoidoverwritingbuiltincmdlets?view=ps-modules))

There is already a `Stop-Process` cmdlet in both PS5 and PS7. The only way `Stop-Process` is used in other scripts is by passing it a single process ID or a `Process` object. Both are supported by the built-in cmdlet so this one is not needed.

Reply:
Just checked the other changes - so instead of
`Stop-Processes -Name <list of process names>`
we should have
`Get-ProcessId -Name <list of process names> | Stop-Process`
right?

Btw I'd like to note that our original intention was to be able to kill a process:
1. By name, regardless of command line.
2. By name and command line.
3. By command line, regardless of name.

AFAIK 3. is not covered by the changes, because the name parameter is mandatory, but we don't actually need that in our TC builds anymore.

Reply:
The built-in `Stop-Process` already stops multiple processes. You can provide multiple program names, process IDs or `Process` objects to it.
![image](https://user-images.githubusercontent.com/4007293/179984777-1492f20c-9663-4503-8c63-0f75163a4268.png)

For example this works on Windows Powershell (just verified):
```powershell
Stop-Process -Name node, bash, python
```

If you want to kill multiple processes by name and command line too, that would be:
```powershell
$nameAndCommandLine = @{ Name = "node" }, @{ Name = "dotnet"; CommandLine = "SomeApp" }
$nameAndCommandLine | % { Get-ProcessId @_ } | Stop-Process
```

Reply:
Oh, I see! The situation was a bit different back when the command line filter was a requirement, so that's why I factored this all out into `Stop-Processes` and I forgot that Stop-Process accepts multiple process names. That's enough for TeamCity now, actually. Thanks for explaining!